lib/pagination.rb

```diff
@@ -1,25 +1,39 @@
 # frozen_string_literal: true
 
 module Pagination
-  def paginated_result(cursor: nil, page_size: nil, order_column: nil)
+  def paginated_result(start_with: nil, page_size: nil, order_column: nil)
     model = @opts[:model]
     page_size = (page_size&.to_i || 10).clamp(1, 100)
```

Comment:
Not to your patch per se, but what is this to_i being run on? to_i is known to be a little...relaxed...in erroneous conditions.

```
[1] clover-development(main)> 'hello'.to_i
=> 0
```

If we only want decimal numbered strings:

```
[3] clover-development(main)> Integer('hello', 10)
ArgumentError: invalid value for Integer(): "hello"
from (pry):3:in `Integer'
```

Reply:
Yes, I've also realized that this weekend. I'll be adding validation for pagination parameters.